src/main/java/com/conveyal/gtfs/graphql/fetchers/JDBCFetcher.java

```diff
@@ -23,18 +25,22 @@
 import java.util.HashSet;
 import java.util.List;
 import java.util.Map;
+import java.util.Objects;
 import java.util.Set;
+import java.util.concurrent.CompletableFuture;
 import java.util.stream.Collectors;
 import java.util.stream.Stream;
 
+import static com.conveyal.gtfs.graphql.GTFSGraphQL.getDataSourceFromContext;
+import static com.conveyal.gtfs.graphql.GTFSGraphQL.getJdbcQueryDataLoaderFromContext;
 import static com.conveyal.gtfs.graphql.GraphQLUtil.multiStringArg;
 import static com.conveyal.gtfs.graphql.GraphQLUtil.stringArg;
 import static graphql.schema.GraphQLFieldDefinition.newFieldDefinition;
 
 /**
  * A generic fetcher to get fields out of an SQL database table.
  */
-public class JDBCFetcher implements DataFetcher<List<Map<String, Object>>> {
+public class JDBCFetcher implements DataFetcher<Object> {
```

Comment:
Do we now have cases where we're fetching something other than lists of string to object maps? I think there are other places where we assume the results of fetches are of this type, and I am hesitant to lose this type information.